docs/source/index.rst

```diff
@@ -56,6 +56,7 @@ xla_model
 .. autofunction:: xla_device
 .. autofunction:: xla_device_hw
 .. autofunction:: is_master_ordinal
+.. autofunction:: xrt_world_size
```

Comment:
why add `xrt_world_size ` here? I thought we are going to use `xr.world_size` instead